### DISCUSSION TITLE
stoicism/morning_ritual/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#6] [General]
# stoicism/morning_ritual/

https://shanenull.com/stoicism/morning_ritual/

**Comment by shane0** [comment]
# reflection

- https://youtube.com/playlist?list=PL0w7f776wBnGkL_NYAPQCXbMBILYR4NcW&si=A1ERh48TQ_0d6gTV
- [ ] [food todo](https://shanenull.com/food/todo/)
- [ ] [2x week 30m meditations](https://forms.gle/Q8dFKBWahbrXH7iS6)
- [ ] [diamond sutra mind seal: subdue the mind abide in nothing](http://shanenull.com/buddhism/2024/mind_seal/)
- [ ] [diamond sutra](https://shanenull.com/buddhism/2024/diamond_sutra/)

## level + journals

- [level 1](https://shanenull.com/buddhism/2023/record/)
- [level 2](https://shanenull.com/buddhism/2024/level_2/)
- [level 3](https://forms.gle/Q8dFKBWahbrXH7iS6)

## meditations

- [seven round compassion](https://shanenull.com/buddhism/2024/seven/)
- [heart sutra memorization](https://shanenull.com/buddhism/2024/heart_sutra_memorization/)
- [diamond sutra timestamp on chapters](https://youtu.be/C85XEdIZIoQ)

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/seven/

**Reply by shane0** [reply]
# unanswerable questions

- https://shanenull.com/buddhism/2023/confusion/#unanswerable-questions

**Reply by shane0** [reply]
https://shanenull.com/buddhism/latest/mind_ground/

**Reply by shane0** [reply]
https://youtube.com/shorts/gjchGqNQW_E?si=U6sVwg24mkpJXnDK

**Comment by shane0** [comment]
diamond sutra at night
http://shanenull.com/buddhism/2024/mind_seal/

**Comment by shane0** [comment]
add todo
add yoga to weekly or daily
https://play.google.com/books/reader?id=IUBxMQAAAEAJ&pg=GBS.PA13

**Comment by shane0** [comment]
http://shanenull.com/buddhism/2024/glossary/#buddha-eyes

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2024/mind_ground/

**Comment by shane0** [comment]
When you have more past than future you learn

**Comment by shane0** [comment]
# earthquake maps

- aka the map aliens use to lacate the dumb humans like me who live on fault lines
- https://glowy-earthquakes.glitch.me/
- more earth cracks 
- https://scitechdaily.com/a-total-surprise-geologists-uncover-new-origin-story-for-deadly-seattle-fault/

**Comment by shane0** [comment]
I f we have a hundred thousand, we want 
a million. If we have a million, we want 
ten million. Having ten million, we want a 
billion; having a billion, we want the entire 
country; having the entire country, we want 
to conquer the world; having conquered the 
world,  we set  our  sights  on outer  space…
Human desire is endless and insatiable. By 
attempting to satisfy it, our lives will be 
immersed in greed, an ocean of suffering 
from beginning to end.

**Comment by shane0** [comment]
https://hazards.fema.gov/nri/map

**Reply by shane0** [reply]
taiwan earthquake
https://earthquake.usgs.gov/earthquakes/eventpage/us7000m9g4/executive

**Comment by shane0** [comment]
enhanced id <https://dol.wa.gov/id-cards/enhanced-id-card-eid/get-enhanced-id-card-eid#:~:text=%247%20to%20%2432-,What%20is%20an%20enhanced%20ID%20(EID)%3F,travel%20within%20the%20United%20States.>

**Comment by shane0** [comment]
audio
https://www.bdkamerica.org/the-teaching-of-buddha-audio-files/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/latest/mind_ground/

**Comment by shane0** [comment]
https://youtu.be/3PyhXlHDkNI?si=wvpyF2xijU0p8Sng
